mig-blue-green/testing/test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+#
+# The end-to-end test against empty sandbox project.
+# Runs all steps of setup/validate/teardown in sequence.
+# 
+#
+# Usage:
+#   bash test.sh
+#
+#
+# TODO: runner @cloudbuild.gserviceaccount.com must own sandbox project
+# MAYDO: avoid check in of sandbox project id to prevent potentiall abuse
+#
+
+set -e
+
+readonly SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+pushd "$SCRIPT_DIR/.."
+
+gcloud builds submit \
```

Comment:
I assume this file is for local testing. Adding this command to the README may be helpful but we usually don't provide a script like this.

Reply:
done